apps/web/hooks/useLinkResolver/useLinkResolver.ts

```diff
@@ -39,6 +39,10 @@ export const routesTemplate = {
     is: '/s/stafraent-island/[slug]',
     en: '',
   },
+  applications: {
+    is: '/umsokn',
+    en: '/en/applications',
```

Comment:
Just as a curiousity.
The icelandic is in singular but the english in plural.

Would it look something like this?
```typescript
is: '/umsoknir/',
en: '/en/applications
```

Reply:
Ah but of course, /umsoknir/ are the applications themselves.
I see.

Reply:
Unfortunately `/umsoknir` is used by the application system (it's a redirect in production) so we can not use it. We asked to do so already. The product owner decided on the singular version.

Reply:
I agree it reads strangely. I can open the topic again with the product owner but we proposed a few options.

Reply:
agree also 👍